iFixit/Bootstrap-Form-Builder
# Input controls:  Add video upload

Dozuki needs a video input control to allow video uploads in data
capture. Pretty self explanatory.

QA:
In order to test this make sure that
the Video Input control appears on Guide Edit with Data Capture enabled.
Also assert that there weren't any regressions. Things like being able
to drag and drop components should still work.

Closes https://github.com/iFixit/dozuki/issues/957
